README.md

```diff
@@ -1,6 +1,6 @@
 <!-- DO NOT ADD frontmatter, this belongs to the GitHub repository only -->
 
-# Welcome to the syslog-ng documentation center source repository
+# Welcome to the {{ site.product.short_name }} documentation center source repository
```

Comment:
As the warning describes above, this file will not be parsed and modified during the building, it is a repo-only readme file, none of these will work